### ISSUE TITLE
Makes Cloudflare protection at kissanime.to loop

### ISSUE THREAD

**Issue by bomjacob** [EFForg/https-everywhere#6218]
http://kissanime.to/
With https everywhere enabled the Cloudflare protection "Please wait 5 seconds" will never go away, and the site cannot be accessed.

**Comment by Foorack** [comment]
**TLDR**: Its not this plugin fault, it is kissanime who have decided to not support https.

This is not a bug with https-everywhere but rather with the kissanime.to site.
From running the network debugger with "Preserve log" and "Disable Cache" both activated when accessing https://kissanime.to shows that the website is responding with a 301 pointing towards the HTTP version of the site. The website does not seem to have any content that can't be loaded over HTTPS. I have already contacted them several days before I saw this post asking them why and if they could enable HTTPS, without response unfortunately.

Could it happen you are running with the "Block all unencrypted requests" option enabled? That would explain why you are stuck in that loop.

Dear Regards,
Foorack, and the rest of the Black Nights :stuck_out_tongue_winking_eye:
